draft-sharabayko-srt.md

```diff
@@ -426,4 +550,33 @@ TODO acknowledge.
 
 # Packet Sequence List coding {#packet-seq-list-coding}
 
+For any single packet sequence number or two consecutive sequence numbers, 
+it uses the original sequence number in the field. The first bit 
+must start with "0".
+
+~~~
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|0|                   Sequence Number                           |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
```

Comment:
> or two consecutive sequence numbers

The first figure is for one sequence number. There might be confusion from the first paragraph.

Maybe this way:
_The receiver notifies the sender about the lost packets using the NAK packets. The NAK packet contains a list of sequence numbers of lost packets. 
To report a loss of one sequence, the first bit must start with "0"..._

Reply:
Just removed `or two consecutive sequence numbers`. Logically, I think we don't need to care about the case as outstanding. 

Another point is that, as I mentioned in a commit message(https://github.com/Haivision/srt-rfc/pull/6/commits/c1c4de61544d400bdbc825b7d5e1cbc86785fb6e),  the list coding is a very general way so I tend not to mention 'Lost' or 'Loss'  in the appendix section.